package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "lcdgame",
+  "version": "0.3.4",
+  "description": "LCD game library in html5/javascript, to recreate simulations of 80s Game & Watch by Nintendo and other electronic handheld games like Tandy, Sunwing, Mini Arcade etc.",
+  "main": "bin/lcdgame.js",
+  "bin": {
+    "lcdgame": "lcdgame - kopie (2).js"
+  },
+  "scripts": {
+    "minify": "uglifyjs bin/lcdgame.js --comments /^!/ --source-map --mangle --output bin/lcdgame.min.js",
+    "serve:local": "serve ./",
```

Comment:
replaces `run in chrome.bat`. Will server the current directory, so that it can be opened in any browser from localhost.